doc/Rules.md

```diff
@@ -992,6 +992,30 @@ This rule supports 0-prefixing ordered list items for uniform indentation:
 ...
 ```
 
+Note: This rule will report violations for cases like the following where an improperly-indented code block (or similar) appears between two list items and "breaks" the list in two:
+
+~~~markdown
+1. First list
+
+```text
+Code block
+```
+
+1. Second list
+~~~
+
+The fix is to indent the code block so it becomes part of the preceding list item as intended:
+
+~~~markdown
+1. First list
+
+   ```text
+   Code block
+   ```
+
+1. Still first list
```

Comment:
Tiny request: Please change this 1 to a 2 to make it a little clearer this is the second item of the same list.